app/lang/i18n.en.json

```diff
@@ -12,6 +12,7 @@
   "GridActionYoutube": "YouTube action",
   "GridActionPodcast": "Podcast action",
   "GridActionChangeLang": "Change content language / voice",
+  "GridActionVocabLevelToggle": "Explore Vocabulary",
```

Comment:
i wonder if we should do a more generic action name here in order to have space for extensions in the future, but I'll ask Manuela in a separate comment about it.